deploy/helm/kubecf/assets/operations/instance_groups/diego-cell.yaml

```diff
@@ -94,6 +143,11 @@
 - type: replace
   path: /instance_groups/name=diego-cell/jobs/name=rep/properties/quarks?
   value:
+    # Disable the ephemeral rep disk
+    bpm:
+      processes:
+      - name: rep
+        ephemeral_disk: false
```

Comment:
I'm not sure why this is needed. I assume is is necessary to avoid the operator from provisioning an *additional* volume because you already defined one earlier via the `env/agent/settings/disks` modifications.

If there is another reason, please let me know!